packages/storefront-lwc/src/modules/commerce/header/header.scss

```diff
@@ -1,3 +1,8 @@
+@import "../../../scss/variables";
```

Comment:
we should avoid this one as it will be deprecated

Reply:
This import has been removed.